### DISCUSSION TITLE
Link #3 in checklist is broken

### DISCUSSION THREAD

**Discussion by liquidbear99** [zwave-js/zwave-js#2356] [Ideas] [answered]
🚨🚨🚨 STOP! STOP! STOP! 🚨🚨🚨

  Before opening an issue, please read and follow these steps:

  1. Are you using HomeAssistant?
  If yes, please open your issue at https://github.com/home-assistant/core/issues
  UNLESS a developer told you to come here.

  2. Are you using ZWaveJS2MQTT?
  If yes, please open your issue at https://github.com/zwave-js/zwavejs2mqtt/issues
  UNLESS a developer told you to come here.

  **3. Check the troubleshooting section if your problem is described there:
  https://zwave-js.github.io/node-zwave-js/#/development/troubleshooting**

  4. Check the changelog if your problem was already fixed recently.
  https://github.com/zwave-js/node-zwave-js/blob/master/CHANGELOG.md
  We cannot provide support if you are not using the latest version.

  🙏🏻🙏🏻🙏🏻 Thanks, now onto your issue:

**Comment by AlCalzone** [comment, marked as answer]
Thanks!
